style.css

```diff
@@ -1,7 +1,396 @@
+}
+
+@media screen and (max-width: 320px) {
+  .header { 
+    background-image: url(images/nav/pic-bgr.jpg);
+    background-size: cover;
+    max-height: 334px;
+    padding: 0 calc(30px - 2 * (30px - (30 * 100% / 320))); 
+  }
+
+  .nav {
+    display: flex;
+    justify-content: flex-end;
+    margin-bottom: 15px;
+    padding-top: 33px;
+  }
+
+  .nav__link { display: none; }
+
+  .burger { display: block; }
+
+  .introduce {
+    color: #e1e6e9;
+    display: flex;
+    flex-direction: column;
+    font-weight: 900;
+    justify-content: center;
+    padding-bottom: 41px;
+  }
+
+  .introduce__title {
+    align-self: center;
+    font-size: 30px;
+    margin-bottom: 5px;
+    max-width: 263px;
+    text-align: center;
+  }
+
+  .introduce__paragraph {
+    align-self: center;
+    font-size: 15px;
+    margin: 0 0 109px 0;
+    max-width: 212px;
+    text-align: center;
+  }
+
+  .button {
+    background-color: #ffda45;
+    border: 0;
+    box-shadow: 5px 5px 20px rgba(0, 0, 0, .25);
+    font-size: 20px;
+    font-style: normal;
+    font-weight: 900;
+    height: 45px;
+    outline: none;
+    text-align: center;
+    width: 260px;
+  }
+
+  .button_mediumvioletred {
+    align-self: center;
+    background-color: #e0036f;
+    color: #fff;
+    font-size: 15px;
+  }
+
+  .tickets {
+    align-items: center;
+    background-color: #e3eafe;
+    display: flex;
+    flex-direction: column;
+    padding-top: 23px;
+    padding-bottom: 23px;
+  }
+
+  .title {
+    color: #585858;
+    font-weight: 500;
+    font-size: 20px;
+  }
+
+  .tickets__title {
+    margin: 0;
+    margin-bottom: 7px;
+    text-align: center;
+    max-width: 250px;
+    max-height: 92px;
+  }
+
+  .tickets__paragraph {
+    color: #263b83;
+    font-weight: 900;
+    font-size: 20px;
+    margin: 0 0 19px 0;
+  }
+
+  .tickets__button { margin-bottom: 28px; }
+
+  .tickets__reserve {
+    background-color: #e3eafe;
+    border-bottom: 3px solid #ffda45;
+    border-left: 0;
+    border-right: 0;
+    border-top: 0;
+    color: #585858;
+    font-size: 20px;
+    font-weight: 900;
+    outline: none;
+    padding-bottom: 13px;
+  }
+
+  .participants {
+    align-items: center;
+    background-color: #405dbb;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-bottom: 34px;
+    max-height: 442px;
+    max-width: 320px;
+  }
+
+  .wrap-info {
+    color: #fff;
+    font-weight: 900;
+    margin: 0;
+    padding-top: 37px;
+    padding-left: 0;
+  }
+
+  .wrap-info__title {
+    align-self: center;
+    font-size: 20px;
+    margin: 0 0 4px 0;
+    max-width: 152px;
+    text-align: center;
+  }
+
+  .wrap-info__paragraph {
+    align-self: center;
+    font-size: 10px;
+    padding: 0;
+    margin: 0 0 21px 0;
+    max-width: 215px;
+    text-align: center;
+  }
+
+  .wrap-info__pic {
+    align-self: center;
+    max-height: 222px;
+    max-width: 260px;
+    padding-left: 30px;
+    padding-right: 30px;
+  }
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/IFW.2/pullRequest?prid=6289680)